AUTHORS

```diff
@@ -0,0 +1,1197 @@
+# Authors ordered by first contribution.
+
+Sascha Depold <sascha@depold.com>
+Israel De La Hoz <israel.delahoz@koombea.com>
+Vangelis Tsoumenis <kioopi@gmail.com>
+Allan Carroll <allanca@gmail.com>
+corpix <me@corpix.ru>
+Mark Kornfilt <mark.kornfilt@livestream.com>
+Laurent Zuijdwijk <Laurent.Zuijdwijk@gamil.com>
+Meg Sharkey <meg@metamarketsgroup.com>
```

Comment:
Gosh. You know how old I've been when I worked with Meg?

Reply:
@megshark nostalgic 👋  from me

Same to @mickhansen and @janmeier

Reply:
@sdepold I'm almost in tears seeing what progress you've start so far, thank you for your hard work!

Reply:
Yeah, good times, good times! I miss it :wave: